check_ruby.sh

```diff
@@ -44,7 +44,7 @@ pushd "${PROJECT_REPO}" > /dev/null || exit 1
 
     if [ -f "${script_file}" ] &&
        ( [[ ${script_file} == *.rb ]] ||
-      grep '^#!/bin/.*ruby' "${script_file}" ); then
+        (head -1 "$script_file" | grep -q -E '^#!(/usr)?/bin/.*ruby')); then
```

Comment:
Why not let `file` do the work here for us?

```suggestion
         if $(file "$script_file") | grep -e 'Ruby script'; then
```